src/support/json.h

```diff
@@ -257,12 +257,26 @@ struct Value {
     while (*curr && is_json_space(*curr))                                      \
       curr++;                                                                  \
   }
+#define skip_escaped_characters(ptr)                                           \
+  while (*ptr && *ptr != '"') {                                                \
+    if (*ptr == '\\' && *(ptr + 1)) {                                          \
+      ptr++;                                                                   \
+    }                                                                          \
+    ptr++;                                                                     \
+  }
+#define RUNTIME_ASSERT(condition)                                              \
+  if (!(condition)) {                                                          \
+    std::cerr << "Assertion failed: " #condition << " at " << __FILE__ << ":"  \
+              << __LINE__ << "\n";                                             \
+    std::terminate();                                                          \
```

Comment:
We have Fatal for this in `utilities.h`, so we can include that and then replace these two lines with `Fatal() << "..message..";`

Also I think it is best to avoid a C macro for this, and just do `if (..) { Fatal() ..` in the code (the downside of a macro is it isn't clear from the name if it is just an assert - and hence disabled in non-assertion builds - etc.)

Reply:
done.